BUILD.md

```diff
@@ -0,0 +1,13 @@
+# Build Setup
+
+## Prerequisites for Gradle
+1. Clone this repository
+1. Install [Java Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
```

Comment:
v11+

Reply:
I believe the current annotation-based validator is actually JDK 8+.